tests/test_codebase_graph.py

```diff
@@ -81,13 +85,16 @@ def test_should_generate_gml_file_to_codebase_graph(mock_write_gml):
     assert mock_write_gml.called
 
 
-#added to keep compatibility with py2 and py3 (since the dict order are different in different versions)
+# added to keep compatibility with py2 and py3 (since the dict order are different in different versions)
 def find_edge(all_edges, wanted_edge):
     for edge in all_edges:
-        if edge[0] == wanted_edge[0] or edge[0] == wanted_edge[1] and edge[1] == wanted_edge[0] or edge[1] == wanted_edge[1]:
+        found_node_one = edge[0] == wanted_edge[0] or edge[0] == wanted_edge[1]
+        found_node_two = edge[1] == wanted_edge[0] or edge[1] == wanted_edge[1]
+        if found_node_one and found_node_two:
```

Comment:
This could be simplified to just `return found_node_one and found_nod_two`, since if it's `True` it returns `True` and if it's `False` it returns `False`.

Reply:
I don't think we can because it will return the first "answer" and the goal is to iterate over the list until finding the node. 🤔

Reply:
Yep, sorry, missed that!